parquet-column/src/main/java/org/apache/parquet/column/page/PageWriter.java

```diff
@@ -86,4 +86,15 @@ void writePageV2(
    */
   String memUsageString(String prefix);
 
+  /**
+   * Reset the page writer. Reset/reallocate any resources.
+   */
+  public void reset();
```

Comment:
All of the implementations were just empty and never referenced, they have been removed.